server.js

```diff
@@ -69,24 +82,41 @@ app.get('/users/add', (req, res) => {
     firstname, lastname, email, username, password, type,
   };
   const insertUser = 'INSERT INTO users SET ?';
-  // eslint-disable-next-line consistent-return
   con.query(insertUser, body, (err) => {
     if (err) {
       return res.send(err);
     }
-    res.send('successfully added user');
+    return res.send('successfully added user');
   });
 });
 
-/* fake data for backend work */
-app.get('/api/events', (req, res) => {
-  const events = [
-    { id: 1, firstName: 'John', lastName: 'Doe' },
-    { id: 2, firstName: 'Joe', lastName: 'Swanson' },
-    { id: 3, firstName: 'Kanye', lastName: 'West' },
-  ];
+/* adds event to events table */
+app.get('/events/add', (req, res) => {
```

Comment:
I did this in my branch but mine is a post, and I don't think this would work because the column names in the table need to match the variable names here, so teamName would have to be team_name, if you want to use "INSERT into [table] SET ?".